src/dnaio/pairedend.py

```diff
@@ -66,19 +65,7 @@ def __iter__(self) -> Iterator[Tuple[Sequence, Sequence]]:
             name_matcher = record_names_match_bytes
         else:
             name_matcher = record_names_match  # type: ignore
-        for r1, r2 in itertools.zip_longest(self.reader1, self.reader2):
```

Comment:
The comment above ("Avoid usage of zip()") needs to be updated as well (I think it can be deleted entirely).